vembrane/modules/annotate.py

```diff
@@ -41,6 +41,14 @@ def add_subcommmand(subparsers):
         choices=["vcf", "bcf", "uncompressed-bcf"],
         help="Output format.",
     )
+    parser.add_argument(
+        "--backend",
+        "-b",
+        default="pysam",
+        type=Backend.from_string,
+        choices=[Backend.pysam, Backend.cyvcf2],
```

Comment:
... and then also swap the order of these two (which also happens to be alphabetical then)

Reply:
cyvcf2 is default now and order correct